spec/unit/libraries/xcode_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+include MacOS::Xcode
+
+describe MacOS::Xcode::Simulator do
+  context 'when provided an available list of simulators' do
```

Comment:
Do we need to have a similar test for testing if a particular simulator is installed?